libafl_frida/src/helper.rs

```diff
@@ -429,6 +439,7 @@ where
             .map(PathBuf::from)
             .collect::<Vec<_>>();
         FridaInstrumentationHelper::builder()
+            .load_script("/tmp/script.js")
```

Comment:
This really needs to be an option.

Reply:
Of course, it needs to come from the command line or similar.